docker/createManifest.ps1

```diff
@@ -0,0 +1,38 @@
+# Used to create a container manifest.
```

Comment:
It would be good to leave a comment here that says that before running this script one needs to login to $ContainerRegistry using 'docker login ...' command.